src/config/ThemeProvider.tsx

```diff
@@ -76,6 +81,7 @@ export default class ThemeProvider extends React.Component<
   updateTheme = (updates: RecursivePartial<FullTheme>) => {
     this.setState(({ theme }) => ({
       theme: deepmerge(theme, updates),
+      update: true,
```

Comment:
I don't understand the use of update and setting it true here. Are we saying that if you run `updateTheme` then changing the `theme` prop will stop working? How about we just go without the update variable and just update the `theme` as it is passed in.